mysite/code/services/AddonBuilder.php

```diff
@@ -91,8 +90,8 @@ private function buildReadme(Addon $addon, $path) {
 					return;
 				}
 
-				$parser = new MarkdownParser();
-				$readme = $parser->transformMarkdown(file_get_contents($path));
+				$parser = GitHubMarkdownService::create();
+				$readme = $parser->toHtml(file_get_contents($path));
```

Comment:
if the parser fails (API timeout, rate limit, GH down, whatever) then we're going to override all the readmes with empty string.

Can we attempt to catch that and no override the readme?